library/src/main/kotlin/com/livefront/gsonkotlinadapter/util/Gson.kt

```diff
@@ -0,0 +1,12 @@
+package com.livefront.gsonkotlinadapter.util
+
+import com.google.gson.Gson
+
+/**
+ * Helper method that warms the cache for all [classes] passed in.
+ *
+ * This can be used to proactively build the adapters associated the `classes`.
```

Comment:
I think you might be missing a `with`?

```suggestion
 * This can be used to proactively build the adapters associated with the `classes`.
```